README.md

```diff
@@ -20,28 +34,26 @@ contains the most up-to-date values of the signals variables, as of a given
 time.
 
 By convention, functions in the `epiprocess` package that operate on `epi_df`
-objects begin with `epi`. For example: 
+objects begin with `epi`. For example:
 
-- `epi_slide()`, for iteratively applying a custom computation to a variable in
-  an `epi_df` object over sliding windows in time;
-  
-- `epi_cor()`, for computing lagged correlations between variables in an
-  `epi_df` object, (allowing for grouping by geo value, time value, or any other
-  variables).
+-   `epi_slide()`, for iteratively applying a custom computation to a variable in
+    an `epi_df` object over sliding windows in time;
+-   `epi_cor()`, for computing lagged correlations between variables in an
+    `epi_df` object, (allowing for grouping by geo value, time value, or any other
+    variables).
 
```

Comment:
Q: does additional space between `-` and following non-whitespace character change/improve appearance?

Reply:
It doesn't change appearance, just something my .md formatter prefers.